pkg/hope/hypervisors/esxi_hypervisor.go

```diff
@@ -272,3 +270,26 @@ func (hyp *EsxiHypervisor) StartVM(name string) error {
 func (hyp *EsxiHypervisor) StopVM(name string) error {
 	return esxi.PowerOffVmNamed(hyp.node.ConnectionString(), name)
 }
+
+func (hyp *EsxiHypervisor) renderedPackerSpec(vms hope.VMs, vmImageSpec hope.VMImageSpec) (*packer.JsonSpec, error) {
```

Comment:
A block like this also appears in `CreateImage`. Would make sense for this to be extracted from there too, just to improve consistency.

Could also me merged in advance to make this diff less substantial.

Reply:
I'm going to call this one a "wont do". It feels like a substantial change that's more of a "really should redo esxi entirely" than aligned with this PR.